### PR TITLE
BuildingMindsTwinCore (certified-connectors) 

### PR DESCRIPTION
**Pull request by duncanbinger**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/BuildingMindsTwin/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,2128 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "BuildingMinds DigitalTwin Core",
+    "version": "1.0",
+    "description": "Connector to BuildingMinds DigitalTwin Core",
```
Comment (sriyen-msft) [inline certified-connectors/BuildingMindsTwin/apiDefinition.swagger.json:6]: @duncanbinger Please provide a valid and more detailed description for your connector.

File: certified-connectors/BuildingMindsTwin/apiProperties.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            "refreshUrl": {
+              "value": "https://buildingminds.b2clogin.com/buildingminds.onmicrosoft.com/B2C_1A_SignInUnified/oauth2/v2.0/token"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#428e92",
+    "capabilities": [],
+    "publisher": "BuildingMinds",
```
Comment (sriyen-msft) [inline certified-connectors/BuildingMindsTwin/apiProperties.json:34]: @duncanbinger Does your company name spells with out a space between 'Building' and 'Minds' ?

File: certified-connectors/BuildingMindsTwin/apiProperties.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "required",
+          "scopes": [
+            "https://buildingminds.onmicrosoft.com/cd1b0fa3-bc0f-4cd7-a09c-dbaf6c19db40/user_impersonation openid offline_access"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "redirectUrl": "https://global-test.consent.azure-apim.net/redirect",
```
Comment (sriyen-msft) [inline certified-connectors/BuildingMindsTwin/apiProperties.json:14]: @duncanbinger You need to provide only one 'redirectURL' and that should not be test one.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
@duncanbinger Please sign CLA and remove other connector files from your PR. And the icon.png is not required in the PR, please remove it.

**Comment by duncanbinger** [comment]
Files now removed and license agreement signed.

File: certified-connectors/BuildingMindsTwin/apiProperties.json (hunk 2 above)
Comment (duncanbinger) [reply certified-connectors/BuildingMindsTwin/apiProperties.json:33]: @sriyen-msft - Correct. The company name is without a space. BuildingMinds. The legal entity is BuildingMinds GmbH

File: certified-connectors/BuildingMindsTwin/apiProperties.json (hunk 3 above)
Comment (duncanbinger) [reply certified-connectors/BuildingMindsTwin/apiProperties.json]: @sriyen-msft - I have removed the test redirect URI.

File: certified-connectors/BuildingMindsTwin/apiDefinition.swagger.json (hunk 1 above)
Comment (duncanbinger) [reply certified-connectors/BuildingMindsTwin/apiDefinition.swagger.json]: @sriyen-msft description updated accordingly.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**